kinetics_MO.py

```diff
@@ -0,0 +1,261 @@
+import numpy as np
+import pandas as pd
+import pyomo.environ as pyo
+from measure_optimize import (
+    MeasurementOptimizer,
+    SensitivityData,
+    MeasurementData,
+    CovarianceStructure,
+    ObjectiveLib,
+)
+import pickle
+import time
+
+
+# set up problem formulation
+
+# number of time points for DCM
+Nt = 8
+
+# maximum manual measurement number for each measurement
+max_manual_num = 10
+# minimal measurement interval
+min_interval_num = 10.0
+# maximum manual measurement number for all measurements
+total_max_manual_num = 10
+
+# index of columns of SCM and DCM in Q
+static_ind = [0, 1, 2]
```

Comment:
Please add more comments. What are the columns in the CSV file?

Reply:
added.